src/main/java/dev/sigstore/fulcio/client/SigningCertificate.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2022 The Sigstore Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package dev.sigstore.fulcio.client;
+
+import com.google.api.client.util.PemReader;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.gson.Gson;
+import com.google.gson.JsonParseException;
+import dev.sigstore.json.GsonSupplier;
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.security.cert.*;
+import java.util.ArrayList;
+import java.util.Base64;
+import org.conscrypt.ct.DigitallySigned;
+import org.conscrypt.ct.SerializationException;
+import org.conscrypt.ct.SignedCertificateTimestamp;
+
+/** Response from Fulcio that includes a certPath and an SCT */
+public class SigningCertificate {
+
+  private final CertPath certPath;
+  private final SignedCertificateTimestamp sct;
+
+  static SigningCertificate newSigningCertificate(String certs, String sctHeader)
+      throws CertificateException, IOException, SerializationException {
+    CertPath certPath = decodeCerts(certs);
+    SignedCertificateTimestamp sct = decodeSCT(sctHeader);
+    return new SigningCertificate(certPath, sct);
+  }
+
+  @VisibleForTesting
+  static CertPath decodeCerts(String content) throws CertificateException, IOException {
+    PemReader pemReader = new PemReader(new StringReader(content));
```

Comment:
Does this fail if a DER encoded certificate is provided? (I think it should)

Reply:
actually, the request sets the header to only accept application/pem-certificate-chain, I mean it still fails, so I could write a test to define this behavior, but it would be weird if we got to that point.

Reply:
hrmm, maybe that accept type doesn't actually mean anything... I've added the test anyway